src/core/plugins/json-schema-2020-12-samples/fn/types/integer.js

```diff
@@ -5,6 +5,7 @@ import { integer as randomInteger } from "../core/random"
 import formatAPI from "../api/formatAPI"
 import int32Generator from "../generators/int32"
 import int64Generator from "../generators/int64"
+import { applyNumberConstraints } from "./number"
```

Comment:
The behaviour of constraints should be the same for `number` and `integer`, so I used the existing function for `number`.
Should we perhaps move the function out of `number` to some utils/helpers?

Reply:
No need to extract. What you did is completely fine.